packages/block-editor/src/components/block-list/block.js

```diff
@@ -53,10 +53,18 @@ function mergeWrapperProps( propsA, propsB ) {
 		...propsB,
 	};
 
-	if ( propsA?.className && propsB?.className ) {
+	// May be set to undefined, so check if the property is set!
+	if (
+		propsA?.hasOwnProperty( 'className' ) &&
+		propsB?.hasOwnProperty( 'className' )
+	) {
 		newProps.className = classnames( propsA.className, propsB.className );
 	}
-	if ( propsA?.style && propsB?.style ) {
+
+	if (
+		propsA?.hasOwnProperty( 'style' ) &&
+		propsB?.hasOwnProperty( 'style' )
+	) {
```

Comment:
This merges in a fix that will land in https://github.com/WordPress/gutenberg/pull/56912 to ensure classnames are output correctly. Just including it here for now so that this PR will work properly in manual testing 🙂